var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -263,3 +267,49 @@ def cmake_args(self):
             self.set_cray_login_node_cmake_options(spec, cmake_args)
 
         return cmake_args
+
+    def setup_environment(self, spack_env, run_env):
+        """Set up the compile and runtime environments for a package."""
+
+        # Environment settings for cbtf-krell
+        run_env.prepend_path('PATH', self.prefix.bin)
+        run_env.prepend_path('PATH', self.prefix.sbin)
+
+        # Find openspeedshop library path
+        cbtfk_libdir = find_libraries('libcbtf-core.so.1.1.0',
+                                      root=self.prefix,
+                                      shared=True, recursive=True)
```

Comment:
Does this work? We typically search for `libcbtf-core` and it should automatically add `.so` if it's `shared=True`.

Reply:
Fixed this to libcbtf-cbtf only.